### ISSUE TITLE
[A11Y] Form elements must have labels

### ISSUE THREAD

**Issue by jaridnft** [vbertoletti/aloha-website#2]
Fix at least one (1) of these issues:

- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- Element has no title attribute or the title attribute is empty
